### PR TITLE
Fix german translations

### PR DESCRIPTION
**Pull request by Jean28518**

### CODE REVIEW

**Comment by Jean28518** [comment]
Warning: This PR is based on #303.
Are the other commits automatically detected later by merging or should i rebase this at upstream?

**Review by DasCapschen — APPROVED** [review]
Looks okay. Let's see if @HaSa1002 is happy with #303 and then we can merge them both

File: src/addons/jean28518.jTools/jList/jList.tscn [hunk 1]
```diff
@@ -126,6 +126,53 @@ __meta__ = {
...
+}
+
+[node name="Label" type="RichTextLabel" parent="ConfirmationDialog"]
+anchor_right = 1.0
+anchor_bottom = 1.0
+margin_left = 13.0
+margin_top = 22.0
+margin_right = -11.0
+margin_bottom = -44.0
+text = "Lorem ipsum..."
```
Comment (DasCapschen) [inline src/addons/jean28518.jTools/jList/jList.tscn]: dolor sit amet?

**Comment by DasCapschen** [comment]
Please resolve the conflicts, then we can merge it :)

**Comment by Jean28518** [comment]
Should be done :)

**Pull request merged**